From 9484b6ca1be77a69b7675b96c1cf645cffcbbf2e Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Tue, 18 Dec 2012 16:04:01 -0500
Subject: [PATCH 24/24] Ticket 541 - need to set plugin as off in ldif template

Reviewed by: richm(Thanks)
(cherry picked from commit 1f7195980fa7073f88ba04fce74b62b07e43e143)
(cherry picked from commit d324d6782f74b7e28f7a457428fe43c77b39944c)
---
 .../src/scripts/50rootdnaccesscontrolplugin.ldif   |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ldap/admin/src/scripts/50rootdnaccesscontrolplugin.ldif b/ldap/admin/src/scripts/50rootdnaccesscontrolplugin.ldif
index 1b5e8b0..74e23f4 100644
--- a/ldap/admin/src/scripts/50rootdnaccesscontrolplugin.ldif
+++ b/ldap/admin/src/scripts/50rootdnaccesscontrolplugin.ldif
@@ -6,7 +6,7 @@ cn: RootDN Access Control
 nsslapd-pluginpath: librootdn-access-plugin.so
 nsslapd-plugininitfunc: rootdn_init
 nsslapd-plugintype: internalpreoperation
-nsslapd-pluginenabled: on
+nsslapd-pluginenabled: off
 nsslapd-plugin-depends-on-type: database
 # these will be replaced when the server loads the plugin
 nsslapd-pluginId: ID
-- 
1.7.1

